v-next/hardhat/templates/mocha-ethers/test/Lock.ts

```diff
@@ -1,16 +1,19 @@
-import hre from "@ignored/hardhat-vnext";
-import { NetworkConnection } from "@ignored/hardhat-vnext/types/network";
-import { describe, it, before } from "mocha";
+import { network } from "@ignored/hardhat-vnext";
+import { expect } from "chai";
 
-// TODO: Chai as promised support needs to be added either directly
-// or through the updated `chai-matchers` package.
-import * as chai from "chai";
-import chaiAsPromised from "chai-as-promised";
-chai.use(chaiAsPromised);
-const { expect } = chai;
+import "./setup.js";
+
+import { HardhatEthers } from "@ignored/hardhat-vnext-ethers/types";
 
 describe("Lock", function () {
-  let networkConnection: NetworkConnection<"l1">;
+  let networkHelpers: any; // TODO: We need to export this type in @ignored/hardhat-vnext-network-helpers
```

Comment:
We need to export the `NetworkHelpers` as an interface in the plugin. It's now a class, but it should be a quick conversation into an interface. Slightly more involved than just copy and replace, because it has some nested objects.